website/settings.py

```diff
@@ -51,6 +51,13 @@
 SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC = "https://custombuilder.poly.edu/custom_install/xmlrpc/" # Default, currently no repy_v2 support
 #SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC = "https://sensibilityclearinghouse.poly.edu/custominstallerbuilder/xmlrpc/" # SensibilityTestbed's CIB, which supports repy_v2 and thereby NAT traversal.
 
+# In Python 2.7.9+ a HTTPS request to a server using a self-signed certificate
+# will raise an exception.
+# This setting provides to chose whether to accept self-signed certificates
+# (e.g. in DEBUG mode) by using an unverified ssl context.
+# c.f. https://github.com/SeattleTestbed/custominstallerbuilder/issues/16
+UNVERIFIED_SSL_CONTEXT = DEBUG
```

Comment:
I understand, but the hypothetical situation described is for a user that sets UNVERIFIED_SSL_CONTEXT to True independent of DEBUG.  In other words, someone might try to set DEBUG to False, but UNVERIFIED_SSL_CONTEXT to True.

Are you saying that users should not attempt to modify UNVERIFIED_SSL_CONTEXT, and only modify DEBUG?  If so, that should documented somewhere.